### ISSUE TITLE
Error: Plugin name should be specified

### ISSUE THREAD

**Issue by paulmist** [1000ch/Sublime-svgo#10]
Getting a new error.

Everything up-to-date.

```
svgo
Error: /Users/*****/Library/Application Support/Sublime Text 3/Packages/SVGO/node_modules/svgo/lib/svgo/config.js:97
throw Error(`Plugin name should be specified`);
^

Error: Plugin name should be specified
at resolvePluginConfig (/Users/*****/Library/Application Support/Sublime Text 3/Packages/SVGO/node_modules/svgo/lib/svgo/config.js:97:13)
at /Users/*****/Library/Application Support/Sublime Text 3/Packages/SVGO/node_modules/svgo/lib/svgo.js:56:7
at Array.map (<anonymous>)
at optimize (/Users/*****/Library/Application Support/Sublime Text 3/Packages/SVGO/node_modules/svgo/lib/svgo.js:55:37)
at minify (/Users/*****/Library/Application Support/Sublime Text 3/Packages/SVGO/svgo.js:36:10)
at /Users/*****/Library/Application Support/Sublime Text 3/Packages/SVGO/svgo.js:7:17
at processTicksAndRejections (node:internal/process/task_queues:93:5)
```

**Comment by saneef** [comment]
Facing the same problem

**Comment by thecheg** [comment]
same for me :(

**Comment by josfaber** [comment]
It's probably a 9.x thing, as in my case this error came to life when updating imagemin-svgo from 8.0 to 9.0.

Try changing the svgo plugin syntax from this:
```
          [
            'svgo',
            {
              plugins: [
                {
                  removeViewBox: false,
                },
              ],
            },
          ],
```

to this:
```
          [
            'svgo',
            {
              plugins: [
                {
                  name: "removeViewBox",
                  active: false,
                },
              ],
            },
          ],
```

As found on [github](https://github.com/svg/svgo)

**Comment by WinSanPang** [comment]
Having the same problem too. I don't have any SVGO built-in plugins installed either 🤷‍♂️

**Comment by davidwebca** [comment]
Hello! This is basically because this Sublime plugin adds a few svgo plugins by default and svgo 2 changed the way to pass plugins. I'll be making a pull request to fix the issue, but for those who need it right away, go into your package's files and edit svgo.js with the following : 

```js
'use strict';

const getStdin = require('get-stdin');
const { optimize } = require('svgo');

getStdin()
  .then(data => minify(data))
  .then(data => process.stdout.write(data));

function minify(data) {
  const options = JSON.parse(process.argv[2]);
  const svg = Buffer.isBuffer(data) ? data.toString() : data;
  const plugins = [];

  const defaultOptions = [
    {
      name: "removeTitle",
      active: false
    },
    {
      name: "removeViewBox",
      active: false
    }
  ];

  // Add user plugins
  for (const plugin of Object.keys(options.plugins || [])) {
    plugins.push({
      name: options.plugins[plugin].name,
      params: options.plugins[plugin].params
    });
  }

  // Set default options
  for (const option of Object.keys(defaultOptions)) {
    if (!plugins.find(plugin => option in plugin)) {
      plugins.push({
        name: defaultOptions[option].name,
        params: defaultOptions[option].params
      });
    }
  }

  let r = optimize(svg, {
    js2svg: {
      pretty: options.pretty,
      indent: options.indent
    },
    plugins: plugins,
  });

  return Buffer.from(r.data);
}

```

**Comment by 1000ch** [comment]
@paulmist @saneef @thecheg @WinSanPang @davidwebca sorry for bothering you, I believe this seems to be also fixed on the latest version (v4.4.0).
